### PR TITLE
Added documentation on usage of External IP address for egress traffic on azure

### PR DESCRIPTION
**Pull request by whites11**
Towards: https://github.com/giantswarm/giantswarm/issues/18048

Added documentation about a feature to be released in azure release 15.1.0 that allows to use an existing IP address for egress traffic of worker nodes.

### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [x] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/master/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

**Review by T-Kukawka — APPROVED** [review]

**Comment by T-Kukawka** [comment]
U rock dude!

File: src/content/advanced/egress-ip-address-azure/index.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+metadata:
+  name: ab123
+  ...
+  annotations:
+    giantswarm.io/workers-egress-external-public-ip: "/subscriptions/<subscription ID>/resourceGroups/<resource group>/providers/Microsoft.Network/publicIPAddresses/<public ip name>"
+spec:
+  ...
+```
+
+You can set the annotation to either an existing `Workload Cluster` or a brand-new one.
```
Comment (T-Kukawka) [inline src/content/advanced/egress-ip-address-azure/index.md:46]: This is awesome

**Review by T-Kukawka — APPROVED** [review]
🚀

File: src/content/advanced/egress-ip-address-azure/index.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Caveats and limitations
+
+- You can't use the same Public IP address for more than one `Workload Cluster` at the same time. This is an Azure limitation. 
+- You can enable and disable the use of an External IP Address for a `Workload Cluster` at any time by adding or
+  removing the `annotation` from the `AzureCluster` Custom Resource. Reconciliation takes just a few minutes to switch
+  between the two modes and there shouldn't be any significant network packet loss while switching.
+- This feature is only supported for Node Pools and not for the Master node.
+- If you enable the feature in an existing `Workload Cluster`, the previously used `Public IP` will not be deleted. This can lead to some undesired increase in the Azure bill.
+  If the external IP address feature is disabled again, the same IP address will be reused.
```
Comment (fiunchinho) [inline src/content/advanced/egress-ip-address-azure/index.md:56]: I'd clarify what is "the same IP" here. The default Public IP created by our operator?

Comment (whites11) [reply src/content/advanced/egress-ip-address-azure/index.md]: how about now?

**Review by fiunchinho — APPROVED** [review]

**Pull request merged**